common/v2/features/PurchaseMembership/components/MembershipPlanCard.tsx

```diff
@@ -0,0 +1,59 @@
+import React, { useContext } from 'react';
+import styled from 'styled-components';
+
+import { Typography } from 'v2/components';
+import { SPACING, COLORS, BREAK_POINTS } from 'v2/theme';
+
+import { IMembershipConfig, IMembershipId } from '../config';
+import { AssetContext } from 'v2/services/Store';
+import { Asset } from 'v2/types';
+
+const PlanCard = styled.div<{ fullwith?: boolean }>`
+  @media screen and (max-width: ${BREAK_POINTS.SCREEN_SM}) {
+    width: ${props => (props.fullwith ? '100%' : '48%')};
+    margin-top: ${SPACING.SM};
+    flex-direction: ${props => (props.fullwith ? 'row' : 'column')};
+    justify-content: ${props => (props.fullwith ? 'space-evenly' : 'flex-start')};
+    height: ${props => (props.fullwith ? '150px' : '212px')};
+  }
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 18%;
+  padding: ${SPACING.BASE};
+  background-color: ${COLORS.WHITE};
+  border-radius: 6px;
+  box-shadow: 0px 3px 6px rgba(0, 0, 0, 0.07);
+`;
+
+const Description = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  height: 100%;
+  & > * {
+    padding-top: ${SPACING.SM};
+  }
+`;
+
+const STypography = styled(Typography)`
+  color: ${COLORS.BLUE_BRIGHT};
+  font-style: italic;
+  text-align: center;
+`;
+
+export default ({ plan }: { plan: IMembershipConfig }) => {
+  const { assets } = useContext(AssetContext);
+  const planAsset = assets.find(asset => asset.uuid === plan.assetUUID) as Asset;
```

Comment:
We have a method `getAssetByUUID` in helpers so you don't need to reimplement it.